src/service/performance-impl.js

```diff
@@ -257,6 +260,8 @@ export class Performance {
           this.tickDelta('pc', userPerceivedVisualCompletenesssTime);
         });
         this.prerenderComplete_(userPerceivedVisualCompletenesssTime);
+        // Mark this instance in the browser timeline.
+        this.mark('pc');
```

Comment:
Double checking that the use of `mark` here instead of `tick` is intentional. Since `tick` with one argument always calls `mark`, why is it appropriate to use `mark` here, as pc is `tick`-ed elsewhere?

Reply:
I think the logic here is right. The else clause calls `tick` which implicitly calls `mark`. Because `tickDelta` does not, but in this case it makes sense to mark it anyway, it is called manually. Also `tickDelta` is only called later or never (when doc becomes visible) while the mark is immediate at the instance in time where it is appropriate.